plugins/history_plugin/history_plugin.cpp

```diff
@@ -306,6 +308,10 @@ namespace eosio {
             ("filter-out,F", bpo::value<vector<string>>()->composing(),
              "Do not track actions which match receiver:action:actor. Action and Actor both blank excludes all from Reciever. Actor blank excludes all from reciever:action. Receiver may not be blank.")
             ;
+      cfg.add_options()
+            ("history-plugin-prune-timeout-us", bpo::value<int32_t>()->composing(),
+             "Timeout for prune history request (microseconds).")
```

Comment:
See below. Change description to "Timeout for history API calls (microseconds)."